generator/ServiceClientGeneratorLib/Generators/Marshallers/JsonRPCExceptionUnmarshaller.tt

```diff
@@ -37,8 +37,35 @@ namespace <#=this.Config.Namespace #>.Model.Internal.MarshallTransformations
         {
             context.Read();
 
+    <#
+            if  (this.Config.ServiceModel.IsAwsQueryCompatible)
+            {
+#>         
+            string errorCode = errorResponse.Code;
```

Comment:
We do conventionally use `var` for the left side of local variable declaration wherever possible.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/declarations#implicitly-typed-local-variables